eng/Generate.ps1

```diff
@@ -229,9 +229,18 @@ $settings = @{
     'profiles' = [ordered]@{}
 };
 
-foreach ($key in Sort-FileSafe ($swaggerDefinitions.Keys))
+# here we put the source code generation project (map $swaggerDefinitions) and the test code generation project (map $swaggerTestDefinitions) together
+$testProjectEntries = @{};
```

Comment:
nit: does the "testProjectEntries" here include both src and tests entries?   maybe "allProjectEntries" is a better name?